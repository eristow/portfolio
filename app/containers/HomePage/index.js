/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 */

import React from 'react';
import styled from 'styled-components';

import Button from 'components/Button';

const ButtonContainer = styled.div`
  display: flex;
  flex-direction: row;
`;

export default function HomePage() {
  return (
    <div>
      {/* eslint-disable-next-line react/no-unescaped-entities */}
      <h1>Evan Ristow's Portfolio</h1>
      <ButtonContainer>
        <Button type="button">
          <a href="https://www.linkedin.com/in/evan-ristow-1a2477102/">
            LinkedIn
          </a>
        </Button>
        <Button type="button" margin="0px 10px">
          <a href="https://drive.google.com/file/d/1EYDbyOl885N6W-pFKmJajZMro66dCL1D/view?usp=sharing">
            Resume
          </a>
        </Button>
      </ButtonContainer>
    </div>
  );
}
