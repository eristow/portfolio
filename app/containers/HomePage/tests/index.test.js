import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';

import HomePage from '../index';

describe('<HomePage />', () => {
  // TODO: this is erroring out becuase getState() is undefined.
  // Some redux stuff I need to fix for tests
  it.skip('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <Provider store={{}}>
        <IntlProvider locale="en">
          <HomePage />
        </IntlProvider>
      </Provider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
