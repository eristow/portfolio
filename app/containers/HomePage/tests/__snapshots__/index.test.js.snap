// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render and match the snapshot 1`] = `
<div>
  <h1>
    Profiles
  </h1>
  <div
    class="HomePage__ButtonContainer-sc-1js5oki-0 bRyjkW"
  >
    <button
      class="Button-cbu1yy-0 gCZweK"
      color="#03dac6"
      type="button"
    >
      <a
        href="https://www.linkedin.com/in/evan-ristow-1a2477102/"
      >
        LinkedIn
      </a>
    </button>
    <button
      class="Button-cbu1yy-0 hpNGlw"
      color="#03dac6"
      type="button"
    >
      <a
        href="https://drive.google.com/file/d/1EYDbyOl885N6W-pFKmJajZMro66dCL1D/view?usp=sharing"
      >
        Resume
      </a>
    </button>
  </div>
</div>
`;
